pitrery

```diff
@@ -2940,6 +2941,11 @@ case $action in
         info "  $PGDATA/recovery.conf"
         info
 
+        if (( $pgvers > $latest_supported_pg_version )) ; then
+            warn "This version of Postgres is not tested with Pitrery $version!"
+            warn "You may likely require manual tweaks before starting the cluster."
```

Comment:
You may have to add manual steps before starting the cluster.
ou :
It is likely that the cluster needs manual tweaks before starting.